config/initializers/rack_ruby_prof.rb

```diff
@@ -0,0 +1,18 @@
+if ENV['RACK_RUBY_PROF']
+  output = Rails.root.join('tmp/rack_ruby_prof')
+  puts "** Rack::RubyProf is enabled, each request will dump to '#{output}'."
+  puts "** Unset RACK_RUBY_PROF to turn off profiling."
+
+  begin
+    require 'ruby-prof'
+  rescue LoadError
+    puts "Failed! Please make sure ruby-prof is in your Gemfile."
+  else
+    Rails.application.config.middleware.use(
+      Rack::RubyProf,
+      :path        => output,
+      :printers    => {RubyProf::CallStackPrinter => 'call_stack.html'},
```

Comment:
I'd prefer to overwrite first to avoid confusion.  If I want to retain a html, I can rename it.

Here's why... when you run this, every request is profiled, leading to new html files... Just getting to the page I want can sometimes require 3 or more clicks, all of which are profiled requests I don't care about.  With time specific filenames, my directory would be littered with stuff unimportant htmls.

If making the time formatted filenames is helpful in the future, maybe we can make it an option.